rules/typescript.js

```diff
@@ -62,35 +62,42 @@ module.exports = {
         default: {
           memberTypes: [
             'signature',
+
             'public-static-field',
             'protected-static-field',
             'private-static-field',
+
             'public-instance-field',
             'protected-instance-field',
             'private-instance-field',
+
             'public-abstract-field',
             'protected-abstract-field',
-            'private-abstract-field',
```

Comment:
The removed items were removed from the rule since they don't make any sense anyway (private abstract??  how would you inherit that?)